jboss/container/wildfly/launch/keycloak/added/keycloak.sh

```diff
@@ -641,6 +641,22 @@ function configure_security_domain() {
   sed -i "s|<!-- ##KEYCLOAK_SECURITY_DOMAIN## -->|${keycloak_security_domain%$'\n'}|" "${CONFIG_FILE}"
 }
 
+function configure_realms_certificates(){
+   realms_certificates=$1
+   # We need it to be retrieved prior to iterate the web deployments, needed by CLI
+   if [ -n "$realms_certificates" ]; then
+      if [ -n "$token" ]; then
+        # SSO Server 7.0
+        realms_certificates=`$CURL -H "Accept: application/json" -H "Authorization: Bearer ${token}" ${sso_service}/admin/realms/${SSO_REALM} | $(jq '.keys[].certificate') `
```

Comment:
@desmax74 , jq command line is not installed in the EAP image. I suspect that it is installed in RHPAM image right?